doc/Tribler REST API.md

```diff
@@ -36,6 +37,22 @@ Returns an overview of the channel of the user. This includes the name, descript
 }
 ```
 
+## `GET /mychannel/torrents`
+
+Returns a list of torrents in your channel. Each torrent item in the list contains the infohash, name and the timestamp of addition of the torrent.
+
+### Example response
+
+```json
+{
+    "torrents": [{
+        "name": "ubuntu-15.04.iso",
+        "added": 1461840601,
+        "infohash": "e940a7a57294e4c98f62514b32611e38181b6cae"
+    }, ...]
+}
+```
```

Comment:
We should start working on automatic code generation instead of manually maintaining markdown files.